x-pack/test/observability_ai_assistant_api_integration/tests/connectors/connectors.spec.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const observabilityAIAssistantAPIClient = getService('observabilityAIAssistantAPIClient');
+  const supertest = getService('supertest');
+
+  describe('List connectors', () => {
+    it('Returns a 2xx for enterprise license', async () => {
+      let res: any;
+      try {
+        res = await observabilityAIAssistantAPIClient.readUser({
+          endpoint: 'GET /internal/observability_ai_assistant/connectors',
+        });
+      } catch {
+        //
+      }
+
+      expect(!!res).to.be(true);
+    });
```

Comment:
I don't see a value for this test. Is it still a WIP ?
The next test, tests both success and empty list, which means this test adds no value.

Reply:
it's a way to figure out if the request has succeeded. I think I just copied it from a basic license check which does the inverse. I'll find another way for this one.